### PR TITLE
Add the header `x-powered-by: genezio` to uniquely identify genezio

### PR DESCRIPTION
**Pull request by radu1122**
# Pull Request Template

## Type of change

- [x] 🍕 New feature
- [ ] 🐛 Bug Fix
- [ ] 🔥 Breaking change
- [ ] 🧑‍💻 Improvement
- [ ] 📝 Documentation Update

## Checklist

- [x] My code follows the contributor guidelines of this project;
- [x] I have updated the documentation;
- [x] I have added tests;
- [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by andreia-oca** [comment]
This PR should not be pushed before @vladiulianbogdan doesn't approve the dart part. 

I am not sure I covered all the corner cases, especially crons and webhooks.

File: src/bundlers/dart/dartMain.ts [hunk 1]
```diff
@@ -42,8 +41,7 @@ void main() async {
               }
 
               final result = await service.{{name}}();
-              final json = jsonEncode(result);
-              response = '{"jsonrpc": "2.0", "result": $json, "id": 0}';
+              response = '{"jsonrpc": "2.0", "result": result, "id": 0}';
```
Comment (vladiulianbogdan) [inline src/bundlers/dart/dartMain.ts:44]: why did `final json = jsonEncode(result);` disappeared?

Comment (vladiulianbogdan) [inline src/bundlers/dart/dartMain.ts:44]: I am not 100% sure about this change. Did you test it?

Comment (andreia-oca) [reply src/bundlers/dart/dartMain.ts]: I tested (manually) and with integration tests and it seems fine.

Before returning, `response` is going to be encoded in JSON, so IMO it should be fine. [1]

[1] https://github.com/Genez-io/genezio/pull/301/files#diff-b820cfef641b9ca39114a8bc7046ce73e8d8933e6eebfb97ec3831b4f26a7ae1R186

**Pull request merged**